sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -1270,6 +1376,7 @@ class SessionCatalogSuite extends PlanTest {
       }
 
       assert(cause.getMessage.contains("Undefined function: 'undefined_fn'"))
+      catalog.reset()
```

Comment:
here the `SessionCatalog` is instanced with different `conf` parameter.
In `withBasicCatalog`, it just leave it default